api.md

```diff
@@ -96,18 +96,18 @@ The problem is that all of these properties will always be re-created. Instead,
 
 ```jsx
 <mesh visible userData={{ hello: 'world' }} position={[1, 2, 3]} rotation={[Math.PI / 2, 0, 0]}>
-  <sphereGeometry attach="geometry" args={[1, 16, 16]} />
-  <meshStandardMaterial attach="material" color="hotpink" transparent />
+  <sphereGeometry args={[1, 16, 16]} />
+  <meshStandardMaterial color="hotpink" transparent />
 </mesh>
 ```
 
 #### Shortcuts (set)
 
 All properties whose underlying object has a `.set()` method can directly receive the same arguments that `set` would otherwise take. For example [THREE.Color.set](https://threejs.org/docs/index.html#api/en/math/Color.set) can take a color string, so instead of `color={new THREE.Color('hotpink')}` you can simply write `color="hotpink"`. Some `set` methods take multiple arguments, for instance [THREE.Vector3](https://threejs.org/docs/index.html#api/en/math/Vector3.set), give it an array in that case `position={[100, 0, 0]}`.
 
-#### Dealing with non-Object3D's
+#### Attaching and dealing with non-Object3D's
 
-You can put non-Object3D primitives (geometries, materials, etc) into the render tree as well so that they become managed and reactive. They take the same properties they normally would, constructor arguments are passed as an array via `args`. If args change, later on, the object gets re-constructed from scratch! Using the `attach` property objects bind to their parent and are taken off once they unmount.
+Using the `attach` property objects bind to their parent and are taken off once they unmount. You can put non-Object3D primitives (geometries, materials, etc) into the render tree as well, so that they become managed and reactive. They take the same properties they normally would, constructor arguments are passed as an array via `args`. If args changes later on, the object get re-constructed from scratch!
```

Comment:
I think this should be 'change' and 'gets'

`If args changes later on, the object get re-constructed from scratch!`